---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: 'StrainPanDA: Linked Reconstruction of Strain Composition and Gene Content
  Profiles via Pangenome-Based Decomposition of Metagenomic Data'
subtitle: ''
summary: ''
authors:
- Han Hu
- Yuxiang Tan
- Chenhao Li
- Junyu Chen
- Yan Kou
- Zhenjiang Zech Xu
- Yang-Yu Liu
- Yan Tan
- Lei Dai
tags:
- gene content profile
- metagenomics
- microbiome
- pangenome
- strain analysis
categories: []
date: '2022-01-01'
lastmod: 2023-04-06T10:33:47Z
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2023-04-06T12:04:09.089927Z'
publication_types:
- '2'
abstract: Microbial strains of variable functional capacities coexist in microbiomes.
  Current bioinformatics methods of strain analysis cannot provide the direct linkage
  between strain composition and their gene contents from metagenomic data. Here we
  present Strain-level Pangenome Decomposition Analysis (StrainPanDA), a novel method
  that uses the pangenome coverage profile of multiple metagenomic samples to simultaneously
  reconstruct the composition and gene content variation of coexisting strains in
  microbial communities. We systematically validate the accuracy and robustness of
  StrainPanDA using synthetic data sets. To demonstrate the power of gene-centric
  strain profiling, we then apply StrainPanDA to analyze the gut microbiome samples
  of infants, as well as patients treated with fecal microbiota transplantation. We
  show that the linked reconstruction of strain composition and gene content profiles
  is critical for understanding the relationship between microbial adaptation and
  strain-specific functions (e.g., nutrient utilization and pathogenicity). Finally,
  StrainPanDA has minimal requirements for computing resources and can be scaled to
  process multiple species in a community in parallel. In short, StrainPanDA can be
  applied to metagenomic data sets to detect the association between molecular functions
  and microbial/host phenotypes to formulate testable hypotheses and gain novel biological
  insights at the strain or subspecies level.
publication: '*iMeta*'
doi: 10.1002/imt2.41
---
