+++
# Any elements can be added in the body: https://sourcethemes.com/academic/docs/writing-markdown-latex/

widget = "blank"  # See https://sourcethemes.com/academic/docs/page-builder/
headless = true  # This file represents a page section.
active = true  # Activate this widget? true/false
weight = 15  # Order that this section will appear.

title = "ABOUT THE LAB"
subtitle = ""

[design]
  # Choose how many columns the section has. Valid values: 1 or 2.
  columns = "1"

[design.background]
  # Apply a background color, gradient, or image.
  #   Uncomment (by removing `#`) an option to apply it.
  #   Choose a light or dark text color by setting `text_color_light`.
  #   Any HTML color name or Hex value is valid.

  # Background color.
  # color = "navy"

  # Background gradient.
  # gradient_start = "DeepSkyBlue"
  # gradient_end = "SkyBlue"

  # Background image.
  image = "headers/bubbles-wide.jpg"  # Name of image in `static/img/`.
  image_darken = 0.6  # Darken the image? Range 0-1 where 0 is transparent and 1 is opaque.
  image_size = "cover"  #  Options are `cover` (default), `contain`, or `actual` size.
  image_position = "center"  # Options include `left`, `center` (default), or `right`.
  image_parallax = true  # Use a fun parallax-like fixed background effect? true/false

  # Text color (true=light or false=dark).
  text_color_light = true

[design.spacing]
  # Customize the section spacing. Order is top, right, bottom, left.
  padding = ["20px", "0", "20px", "0"]

[advanced]
 # Custom CSS.
 css_style = ""

 # CSS class.
 css_class = ""
+++

Welcome to the Xu lab. Our research focuses on microbiome and host interaction. We apply bioinformatics, microbiology, and animal model to understand how microbiome can be precisely engineered to modulate host phenotypes. Specifically, we are working on the following areas:

**microbiome bioinformatics:** Microbiome data have some idiosyncratic properties (eg sparse and compositional) and require careful treatment. We have been developping scalable and robust bioinformatic tools to mine multi-omic data and discover patterns and knowledge out of them. We are part of the QIIME2 development team and involved in developping multiple tools including scikit-bio, deblur, and calour. These are successfully applied to microbiome studies.

**genomics:** We have developped algorithms to identify novel regulatory ncRNAs and predict their secondary structures. With the ever-increasing microbial genomes available from whole-genomic and metagenomic shotgun sequencing data, there is a bottleneck to understand the function potential of these genomes. We are using deep learning techniques to develop high-quality, consistent annotations for comparative genomics.

**zebrafish mode:** We are using zebrafish as a model to study microbiome & host interaction because of the advantages of zebrafish model, including low cost, scalability, genetic tractability, transparency for imaging, and the available system to analyze complex social behaviors.

We are looking for prospective students and faculty to join us on this exciting research field.
