---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: Multiple-Disease Detection and Classification across Cohorts via Microbiome
  Search
subtitle: ''
summary: ''
authors:
- Xiaoquan Su
- Gongchao Jing
- Zheng Sun
- Lu Liu
- Zhenjiang Xu
- Daniel McDonald
- Zengbin Wang
- Honglei Wang
- Antonio Gonzalez
- Yufeng Zhang
- Shi Huang
- Gavin Huttley
- Rob Knight
- Jian Xu
tags: []
categories: []
date: '2020-04-01'
lastmod: 2023-04-06T10:33:50Z
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2023-04-06T12:04:11.243820Z'
publication_types:
- '2'
abstract: Microbiome-based disease classification depends on well-validated disease-specific
  models or a priori organismal markers. These are lacking for many diseases. Here,
  we present an alternative, search-based strategy for disease detection and classification,
  which detects diseased samples via their outlier novelty versus a database of samples
  from healthy subjects and then compares these to databases of samples from patients.
  Our strategy's precision, sensitivity, and speed outperform model-based approaches.
  In addition, it is more robust to platform heterogeneity and to contamination in
  16S rRNA gene amplicon data sets. This search-based strategy shows promise as an
  important first step in microbiome big-data-based diagnosis. IMPORTANCE Here, we
  present a search-based strategy for disease detection and classification, which
  detects diseased samples via their outlier novelty versus a database of samples
  from healthy subjects and then compares them to databases of samples from patients.
  This approach enables the identification of microbiome states associated with disease
  even in the presence of different cohorts, multiple sequencing platforms, or significant
  contamination.
publication: '*mSystems*'
doi: 10.1128/mSystems.00150-20
---
