---
title: Normalization and Microbial Differential Abundance Strategies Depend upon Data
  Characteristics
_build:
  render: never
  list: never
  publishResources: false
date: '2017-01-01'
publishDate: '2023-04-06T11:50:11.939407Z'
authors:
- Sophie Weiss
- Zhenjiang Zech Xu
- Shyamal Peddada
- Amnon Amir
- Kyle Bittinger
- Antonio Gonzalez
- Catherine Lozupone
- Jesse R. Zaneveld
- Yoshiki VÃ¡zquez-Baeza
- Amanda Birmingham
- Embriette R. Hyde
- Rob Knight
publication_types:
- '2'
abstract: Data from 16S ribosomal RNA (rRNA) amplicon sequencing present challenges
  to ecological and statistical interpretation. In particular, library sizes often
  vary over several ranges of magnitude, and the data contains many zeros. Although
  we are typically interested in comparing relative abundance of taxa in the ecosystem
  of two or more groups, we can only measure the taxon relative abundance in specimens
  obtained from the ecosystems. Because the comparison of taxon relative abundance
  in the specimen is not equivalent to the comparison of taxon relative abundance
  in the ecosystems, this presents a special challenge. Second, because the relative
  abundance of taxa in the specimen (as well as in the ecosystem) sum to 1, these
  are compositional data. Because the compositional data are constrained by the simplex
  (sum to 1) and are not unconstrained in the Euclidean space, many standard methods
  of analysis are not applicable. Here, we evaluate how these challenges impact the
  performance of existing normalization methods and differential abundance analyses.
featured: false
publication: '*Microbiome*'
doi: 10.1186/s40168-017-0237-y
---

