---
title: Optimizing Sequencing Protocols for Leaderboard Metagenomics by Combining Long
  and Short Reads
_build:
  render: never
  list: never
  publishResources: false
date: '2019-10-01'
publishDate: '2023-04-06T11:50:10.673789Z'
authors:
- Jon G. Sanders
- Sergey Nurk
- Rodolfo A. Salido
- Jeremiah Minich
- Zhenjiang Z. Xu
- Qiyun Zhu
- Cameron Martino
- Marcus Fedarko
- Timothy D. Arthur
- Feng Chen
- Brigid S. Boland
- Greg C. Humphrey
- Caitriona Brennan
- Karenina Sanders
- James Gaffney
- Kristen Jepsen
- Mahdieh Khosroheidari
- Cliff Green
- Marlon Liyanage
- Jason W. Dang
- Vanessa V. Phelan
- Robert A. Quinn
- Anton Bankevich
- John T. Chang
- Tariq M. Rana
- Douglas J. Conrad
- William J. Sandborn
- Larry Smarr
- Pieter C. Dorrestein
- Pavel A. Pevzner
- Rob Knight
publication_types:
- '2'
abstract: As metagenomic studies move to increasing numbers of samples, communities
  like the human gut may benefit more from the assembly of abundant microbes in many
  samples, rather than the exhaustive assembly of fewer samples. We term this approach
  leaderboard metagenome sequencing. To explore protocol optimization for leaderboard
  metagenomics in real samples, we introduce a benchmark of library prep and sequencing
  using internal references generated by synthetic long-read technology, allowing
  us to evaluate high-throughput library preparation methods against gold-standard
  reference genomes derived from the samples themselves. We introduce a low-cost protocol
  for high-throughput library preparation and sequencing.
featured: false
publication: '*Genome Biology*'
doi: 10.1186/s13059-019-1834-9
---

